taxi/forms.py

```diff
@@ -50,3 +50,42 @@
         raise ValidationError("Last 5 characters should be digits")
 
     return license_number
+
+
+class DriverSearchForm(forms.Form):
+    username = forms.CharField(
+        max_length=255,
+        required=False,
+        label="",
+        widget=forms.TextInput(
+            attrs={
+                "placeholder": "Search by username",
+            }
+        )
+    )
+
+
+class CarSearchForm(forms.Form):
+    username = forms.CharField(
+        max_length=255,
+        required=False,
+        label="",
+        widget=forms.TextInput(
+            attrs={
+                "placeholder": "Search by model",
+            }
+        )
+    )
+
+
+class ManufacturerSearchForm(forms.Form):
+    username = forms.CharField(
```

Comment:
The field name `username` might not be appropriate for searching manufacturers. Consider renaming it to something more relevant, like `name`.